pkg/platform/runtime/setup/buildlog/messages.go

```diff
@@ -0,0 +1,164 @@
+package buildlog
```

Comment:
I added this file comprising a serialization for all the messages that we can read on the build log streamer websocket connection.  This seemed necessary, as we are now distinguishing between more (and different) message types.

It also increases the legibility of the buildlog.go code IMO.